python/grass/jupyter/__init__.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import
+
+from .init_funcs import *
```

Comment:
To make the grass.jupyter subpackage really work, you will need to add it to makefiles, see the `Makefile` file in `grass/app` or `grass/grassdb` and also add the name to `Makefile` in the parent directory. It should be just copy-pasting, no real Makefile writing is needed, but let me know.